bitnami/aspnet-core/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: aspnet-core
-version: 0.1.4
+version: 0.1.5
```

Comment:
You'll have to bump the minor version instead.

Reply:
Changed.